build.gradle.kts

```diff
@@ -102,35 +102,31 @@ tasks.withType<Detekt>().configureEach {
 }
 
 publishing {
-    publications {
-        create<MavenPublication>("mavenJava") {
-            from(components["java"])
-
-            artifact(javadocJar)
-
-            pom {
-                name.set(LibConfig.name)
-                description.set("Kotlin multiplatform JsonLogic expressions evaluation engine")
-                url.set(LibConfig.repositoryUrl)
-                inceptionYear.set("2022")
-                licenses {
-                    license {
-                        name.set("The Apache License, Version 2.0")
-                        url.set("https://www.apache.org/licenses/LICENSE-2.0.txt")
-                    }
-                }
-                developers {
-                    developer {
-                        name.set("Marek Krogulski")
-                        email.set("marek.krogulski@allegro.pl")
-                    }
+    publications.withType<MavenPublication> {
+        artifact(javadocJar)
+
+        pom {
+            name.set(LibConfig.name)
+            description.set("Kotlin multiplatform JsonLogic expressions evaluation engine")
+            url.set(LibConfig.repositoryUrl)
+            inceptionYear.set("2022")
+            licenses {
+                license {
+                    name.set("The Apache License, Version 2.0")
+                    url.set("https://www.apache.org/licenses/LICENSE-2.0.txt")
                 }
-                scm {
-                    connection.set("scm:svn:${LibConfig.repositoryUrl}")
-                    developerConnection.set("scm:git@github.com:allegro/json-logic-kmp.git")
-                    url.set(LibConfig.repositoryUrl)
+            }
+            developers {
+                developer {
+                    name.set("Marek Krogulski")
```

Comment:
![image](https://user-images.githubusercontent.com/84186160/154922159-3ebf8fdf-a032-48b4-81e8-3cc6c64ddd6a.png)
Do we want to store info only about you or about all of the developers involved?